### ISSUE TITLE
同一张图片不同分辨率的box

### ISSUE THREAD

**Issue by luzeming** [AlexeyAB/Yolo_mark#73]
对于同一张图片不同分辨率可不可以自动调整他的box，时期对应分辨率变化的比例

**Comment by AnthonyZhai** [comment]
@AlexeyAB ,let me translate for you:
He would like to know will the groundtruth bounding box change alongside with the resolution of images?

**Comment by AlexeyAB** [comment]
No. Labels (groundtruth bounding boxes) have relative size [0.0 - 1.0] x [0.0-1.0], so it doesn't depend on image size (resolution).

**Comment by AnthonyZhai** [comment]
@luzeming Labels是相对值，不依赖图片尺寸，会根据公式
$$
x1,y1,w1,h1=从txt读出来的
$$

$$
x = x1 * w , y = y1* h
$$

$$
w = w1 * w, h = h1 * h
$$
就会计算出label的中心点坐标和长宽，这些值是根据图片的size计算的，所以你无论修改图片的分辨率到怎么样，label都会随着变化

**Comment by lc-soft** [comment]
这个问题，在 darknet 的 README.md 中有说明：

![image](https://user-images.githubusercontent.com/1730073/50002711-f2217600-ffdb-11e8-992b-c579fac53bfa.png)
